force-app/main/service/HouseholdNamingService.cls

```diff
@@ -117,27 +131,43 @@ public without sharing class HouseholdNamingService {
     * @return void
     */
     public void updateHouseholdNameAndMemberCount(List<Id> householdOrAccountIds) {
-        if (isHouseholdNamingDisabled) {
+        // Increment total call counter for testing purposes
+        if (Test.isRunningTest()) {
+            testTotalCallCount++;
+        }
+        
+        if (isHouseholdNamingDisabled || isProcessingHouseholdNaming) {
```

Comment:
In theory, using `isProcessingHouseholdNaming` is what recursion control (`TDTM_ProcessControl`) is supposed to do as well, but I wonder if it's not actually working as expected here.

Reply:
This has been removed because it's not necessary